cypress.json

```diff
@@ -0,0 +1,11 @@
+{
+  "baseUrl": "http://localhost:8000",
+  "viewportWidth": 1200,
+  "viewportHeight": 800,
+  "fixturesFolder": "test/cypress/fixtures",
+  "integrationFolder": "test/cypress/integration",
+  "pluginsFile": "test/cypress/plugins",
+  "screenshotsFolder": "test/cypress/screenshots",
+  "supportFile": "test/cypress/support",
+  "videosFolder": "test/cypress/videos"
```

Comment:
Note: @taoeffect by default the tests are created directly at the root `/cypress`, I moved them to `test/` folder to keep consistency. Is that ok for you?

Reply:
Yes all tests should be under the `test/` folder regardless of what framework we're using. :)

Ideally, we would use as little framework-specific folder structure as possible.